src/features/events/l10n/messageIds.ts

```diff
@@ -30,6 +31,11 @@ export default makeMessages('feat.events', {
       '"{eventTitle}" will be restored.'
     ),
   },
+  eventChangeCampaign: {
```

Comment:
I would call this `eventMoveDialog` to correlate with the name of the component.

Reply:
Just a quick comment about my fix for this: 'event move' is already in use as a term for (I think) moving an event to another time slot (see useMoveEvents), so I used the 'change campaign' for my code instead. I just forgot to rename the component.